src/utils/math.js

```diff
@@ -28,11 +28,11 @@
  * Ex1: -2 in the range [0, 10] is interpreted as 0 and thus gives 40 in [40,80]
  * Ex2: 19 in the range [0, 5] is interpreted as 5 and thus gives 80 in [40,80]
  *
- * @param {number} val the value in the source range
- * @param {number} min1 the lower bound of the source range
- * @param {number} max1 the upper bound of the source range
- * @param {number} min2 the lower bound of the target range
- * @param {number} max2 the upper bound of the target range
+ * @param {!number} val the value in the source range
```

Comment:
nit: numbers and other primitives are non-nullable by default, so no need for `!`